test/opcodecnt/basic.txt

```diff
@@ -0,0 +1,43 @@
+;;; TOOL: run-opcodecnt
```

Comment:
I don't blame you for not, there's a lot of boilerplate (and I still don't have it working properly on travis :-))